test/verify/check-pages

```diff
@@ -462,7 +453,12 @@ OnCalendar=daily
         b.wait_visible("#host-apps li a:contains('Services')")
         b.wait_text("#host-apps li a:contains('Services')", "ServicesContains: systemd")
         b.wait_text("#host-apps li a:contains('Services') mark", "systemd")
-        b.assert_pixels("#nav-system", "menu-search")
+        b.assert_pixels("#nav-system", "menu-search", ignore=[".nav-item-status"], skip_layouts=["mobile"])
+        b.set_layout("mobile")
```

Comment:
huh.  This is not what I expected.

I thought the idea was to run all the asserts in all the layouts, automatically, instead of having each test case drive this for itself...

This probably needs some discussion.

Reply:
This is just an exception. Normally all calls to assert_pixels run through all the layouts, but in this case the mobile layout needs some special treatment (clicking on `#nav-system-item`).  Note the `skip_layouts=["mobile"]` in the call to assert_pixels.

Reply:
Gotcha.  Thanks.